cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
+        customView.didTapButton = {
```

Comment:
separate this closure to be a property in the class and use capture self to [weak self] to prevent retain cycles, you can read more about ARC and retain cycles here: 
https://docs.swift.org/swift-book/LanguageGuide/AutomaticReferenceCounting.html

you should mark it with lazy to be able to use self.payment before init the ViewController.
```
lazy var didTapPaymentViewButtonAction = { [weak self] in
	if let payment = self?.payment {
		self?.delegate?.didFinishFlow(amount: payment.amount, currency: payment.currency)
	}
}
```
```
then use paymentView.didTapButton = didTapPaymentViewButtonAction
```